code/Model/UserDefinedForm.php

```diff
@@ -9,6 +9,8 @@
 
 /**
  * @package userforms
+ * @method SilverStripe\ORM\HasManyList<EmailRecipient> EmailRecipients()
+ * @method SilverStripe\ORM\HasManyList<SubmittedForm> Submissions()
```

Comment:
Need `use` statements or FQCN for these models since the relations are added via a trait

Reply:
Having updated the docblock writer, using the FCQN here